jekyll/_cci2/concepts.md

```diff
@@ -212,14 +211,14 @@ jobs:
 # ...
 ```
 
-The Primary Container is defined by the first image listed in [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file. This is where commands are executed. The Docker executor spins up a container with a Docker image. The machine executor spins up a complete Ubuntu virtual machine image. See [Choosing an Executor Type]({{ site.baseurl }}/2.0/executor-types/) document for a comparison table and considerations. Subsequent images can be added to spin up Secondary/Service Containers.
+The primary container is defined by the first image listed in [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file. This is where commands are executed. The Docker executor spins up a container with a Docker image. The machine executor spins up a complete Ubuntu virtual machine image. See [Choosing an Executor Type]({{ site.baseurl }}/2.0/executor-types/) document for a comparison table and considerations. Subsequent images can be added to spin up secondary/service containers.
 
-When using the docker executor and running docker commands, the `setup_remote_docker` key can be used to spin up another docker container in which to run these commands, for added security. For more information see the [Running Docker Commands]({{ site.baseurl }}/2.0/building-docker-images/#accessing-the-remote-docker-environment) guide.
+When using the Docker executor and running docker commands, the `setup_remote_docker` key can be used to spin up another docker container in which to run these commands, for added security. For more information see the [Running Docker Commands]({{ site.baseurl }}/2.0/building-docker-images/#accessing-the-remote-docker-environment) guide.
```

Comment:
Why are we capitalizing "Docker executor" and not "docker commands" and "docker container"?

Reply:
The best thing to do is look at the company's own docs and see if they capitalise it themselves, in this case Docker do. Also we have a marketing style guide that specifies some capitalisations. I'll copy the list to our style guide.... https://docs.google.com/document/d/1dw9AuGVdNXvxOnaU6fDulaPNPaYhSFncUiiDSzBJOR4/edit#heading=h.sksbslz8h1vq

Reply:
Good point! Have now applied capitalization consistently.